packages/neo-one-monitor/src/MonitorBase.js

```diff
@@ -934,10 +1013,16 @@ export default class MonitorBase implements Span {
     name,
     help = 'Placeholder',
     labelNames: labelNamesIn,
+    directReport,
   }: MetricOptions): MetricConstruct {
-    const labelNames = this._getMetricLabelNames(labelNamesIn).map(labelName =>
-      convertMetricLabel(labelName),
-    );
+    let labelNames;
+    if (directReport) {
```

Comment:
Same comment here, don't change this method to do something entirely different with `directReport`, just make a common private method that they both use.